src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -75,7 +75,7 @@ def load_arguments(self, _):
         c.argument('id_', options_list=['--id'], help='Id of the key to be regenerated. Can be found using az appconfig credential list command.')
 
     with self.argument_context('appconfig kv import') as c:
-        c.argument('label', help="Imported KVs and feature flags will be assigned with this label. If no label specified, will assign null label.")
+        c.argument('label', help="Imported KVs and feature flags will be assigned with this label. If omitted, for file and appservice source, will assign null label by default. For AppConfig source, if neither --label nor --preserve-labels is specified, will assign null label by default. If there are duplicate keys in source AppConfig, only the last key will be imported with this label.")
         c.argument('prefix', help="This prefix will be appended to the front of imported keys. Prefix will be ignored for feature flags.")
```

Comment:
nit: For the help message, we don't it to explain every detail or features we added about this argument as cli works as self explain. We will add examples for preserve_label and error out if both label and preserve_label are provided. 
So the help message can be something like "Imported KVs and feature flags will be assigned with this label. If no label specified, will assign null label. Not compatible with preserve_label argument."